modules/nf-core/glimpse2/ligate/environment.yml

```diff
@@ -4,4 +4,4 @@ channels:
   - bioconda
   - defaults
 dependencies:
-  - bioconda::glimpse-bio=2.0.0
+  - bioconda::glimpse-bio=2.0.0=hf340a29_0
```

Comment:
any need to be that strict in the conda env declaration?

Reply:
It was a test to see if it affected the output result.
I was having inconsistency between singularity, docker and conda.
As it it HMM (probabilist statistic), the result change a bit between container (1 instead of 0.999).
I've delete the test and changed it to a file name assertion.